boxtree/translation_classes.py

```diff
@@ -369,12 +374,15 @@ def __call__(self, queue, trav, tree, wait_for=None,
         from_sep_siblings_translation_classes_level_starts = cl.array.to_device(
             queue, from_sep_siblings_translation_classes_level_starts)
 
-        return TranslationClassesInfo(
+        info = TranslationClassesInfo(
                 from_sep_siblings_translation_classes=translation_classes_lists,
                 from_sep_siblings_translation_class_to_distance_vector=distances,
                 from_sep_siblings_translation_classes_level_starts=(
                     from_sep_siblings_translation_classes_level_starts),
-                ).with_queue(None), evt
+                ).with_queue(None)
+
+        info.traversal = trav
```

Comment:
Not a huge fan of stateful updates. Could you pass this in through the constructor?

Reply:
Fixed in 20feb451883327fcfd9c069fdc2439ebde6188b6